core/src/main/scala/pureconfig/DerivedReaders.scala

```diff
@@ -35,6 +36,20 @@ trait DerivedReaders extends DerivedReaders1 {
       def from(value: ConfigValue): Either[ConfigReaderFailures, T] =
         reader.from(value).right.map(unwrapped.wrap)
     }
+
+  // used for tuples
+  implicit def deriveTupleInstance[F: IsTuple, Repr](
+    implicit
+    g: Generic.Aux[F, Repr],
+    gcr: ConfigReader[Repr],
+    pr: ConfigReader[F]): ConfigReader[F] = new ConfigReader[F] {
```

Comment:
I was being a bit lazy and relying on the diverging implicit checker to have access to the `ConfigReader` that reads tuples from `ConfigObject`s. I modified the code as you suggested in f806a5cf7ffd0624a24ece65c462067cb1d8927b.